x-pack/platform/plugins/private/data_visualizer/public/application/common/components/stats_table/data_visualizer_stats_table.tsx

```diff
@@ -303,14 +303,15 @@ const UnmemoizedDataVisualizerTable = <T extends DataVisualizerTableItem>({
                         defaultMessage: 'Hide distributions',
                       })
                 }
+                disableScreenReaderOutput={true}
               >
                 <EuiButtonIcon
                   style={{ marginLeft: 4 }}
                   size={'s'}
                   iconType={!showDistributions ? 'eye' : 'eyeClosed'}
                   onClick={() => toggleShowDistribution()}
                   aria-label={
-                    showDistributions
+                    !showDistributions
```

Comment:
usually the aria-label for the button should match the content of the EuiTooltip